docs/pages/enroll-resources/application-access/okta/app-and-group-sync.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Okta apps and groups sync
+description: Details on Okta integration apps and groups synchronization as Teleport Access Lists
+---
+
+Okta has its own permissions system that doesn't directly map into Teleport. This
+includes permissions granted by Okta groups along with the assignment of users to individual
+applications within Okta. To model this within Teleport, an administrator would typically need
+to carefully craft a labeling system to attach to various Okta apps and groups and a set of
+roles to go along with them.
+
+With the synchronization of Okta apps and groups ass Teleport Access List feature in the Okta
+integration, this work is performed for you. We'll discuss the details of how this works.
+
+## Enrollment
```

Comment:
I'm not sure why the first H2 of this page is a description of other pages. I think this guide would benefit from a better-defined purpose so it doesn't overlap as much with other content.